app/modules/visualization/controllers/visualizationControllers.js

```diff
@@ -59,7 +61,15 @@ angular.module('pcApp.visualization.controllers.visualization', [
 
 
                         }, function (err) {
-                            throw {message: JSON.stringify(err.data)};
+                        	
+                        	var data = {

```

Comment:
this might not be always the reason for the error. You'll need to check if it's 404 or other error. Currently, until https://github.com/policycompass/policycompass/issues/629 is resolved, you can't distinguish the reason.